diff -ur squidwall-0.4e/Makefile squidwall-0.4e.new/Makefile
--- squidwall-0.4e/Makefile	2006-09-02 23:37:23.000000000 +0200
+++ squidwall-0.4e.new/Makefile	2008-04-02 08:59:37.781992367 +0200
@@ -13,8 +13,8 @@
 
 DIET	= $(shell test -x /opt/diet/bin/diet && echo /opt/diet/bin/diet)
 CC	= $(DIET) gcc
-CFLAGS	= -Os -Wall -pipe -I../libowfat
-LDFLAGS	= ../libowfat/libowfat.a
+CFLAGS	= -Os -Wall -pipe -I/usr/include/libowfat/
+LDFLAGS	= /usr/lib/libowfat.a
 
 # extraflags
 CFLAGS += -W
Tylko w squidwall-0.4e.new/: Makefile~
